content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -29,6 +29,10 @@ The developer preview version provides a preview of the upcoming changes to the
 
 - The **Add User To Chat** method ([Web](/messaging/agent-chat-api/v3.4/#add-user-to-chat) & [RTM](/messaging/agent-chat-api/v3.4/rtm-reference/#add-user-to-chat)) always requires an active thread. The `require_active_thread` flag was removed.
 
+### Chat access
+- The **Transfer Chat** method ([Web](/messaging/agent-chat-api/v3.4/#transfer-chat) & [RTM](/messaging/agent-chat-api/v3.4/rtm-reference/#transfer-chat)) always requires an active thread.
+- The **Grant Chat Access** and **Revoke Chat Access** methods were removed. Its use cases are now covered by **Transfer Chat** ([Web](/messaging/agent-chat-api/v3.4/#transfer-chat) & [RTM](/messaging/agent-chat-api/v3.4/rtm-reference/#transfer-chat)). The corresponding pushes, **chat_access_granted** and **chat_access_revoked**, were also removed.
```

Comment:
it's more accurate to say that the corresponding pushes were replaced by chat_access_updated